test/sqlx-promise.js

```diff
@@ -0,0 +1,97 @@
+describe('sqlx-promise', function() {
+
+it('mysql async/await', async () => {
+  child_process.execSync('mysql -uroot < test/mysql_initdb.txt')
+  const client = sqlx.createClient()
+  client.define('*', MYSQL_CONFIG_1)
+
+  const conn = client.getConnection(OPERATER_INFO)
+  let result
+  // insert
+  await conn.insert(
+    'table1',
+    [
+      {a:1, b:21},
+      {a:2, b:22},
+      {a:3, b:23},
+      {a:3, b:123},
+    ])
+  // select
+  result = await conn.select('table1', '*', { $and: { a: 3, b: 123 } })
+  assert.equal(result.rows.length, 1)
+  // update
+  result = await conn.update('table1', {b:1}, {a:3})
+  assert.equal(result.rows.affectedRows, 2)
+  assert.equal(result.rows.changedRows, 2)
+  // selectEx
+  result = await conn.selectEx('table1', 'select * from table1 where a = ?', [2])
+  assert.equal(result.rows.length, 1)
+  assert.equal(result.rows[0].b, 22)
+  // delete
+  result = await conn.delete('table1', {a:3})
+  assert.equal(result.rows.affectedRows, 2)
+  // queryReadonly
```

Comment:
queryReadonly is deprecated ( not listed in README). remove test cases

Reply:
fixed